docs/guides/zookeeper/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: ZooKeeper
+menu:
+  docs_{{ .version }}:
+    identifier: zk-readme-zookeeper
+    name: ZooKeeper
+    parent: zk-zookeeper-guides
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+url: /docs/{{ .version }}/guides/zookeeper/
+aliases:
+  - /docs/{{ .version }}/guides/zookeeper/README/
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+## Supported ZooKeeper Features
+| Features                                                   | Community | Enterprise |
+|------------------------------------------------------------|:---------:|:----------:|
+| Ensemble                                                   | &#10003;  |  &#10003;  |
+| Standalone                                                 | &#10003;  |  &#10003;  |
+| Authentication & Autorization                              | &#10003;  |  &#10003;  |
+| Persistent Volume                                          | &#10003;  |  &#10003;  |
+| Initializing from Snapshot ( [Stash](https://stash.run/) ) | &#10003;  |  &#10003;  |
+| Instant Backup                                             | &#10003;  |  &#10003;  |
```

Comment:
Add/Update like this - 
Backup/Recovery: Instant, Scheduled ( [KubeStash](https://kubestash.com/))
Custom Configuration
Initializing from Snapshot ( [KubeStash](https://kubestash.com/))
TLS: Add ( [Cert Manager]((https://cert-manager.io/docs/)))
Grafana Dashboards
Externally manageable Auth Secret
Reconfigurable Health Checker
Persistent volume